packages/accessibility/src/behaviors/Carousel/carouselBehavior.ts

```diff
@@ -17,11 +25,18 @@ import * as keyboardKey from 'keyboard-key'
 const carouselBehavior: Accessibility<CarouselBehaviorProps> = props => ({
   attributes: {
     root: {
-      role: 'region',
+      role: props.navigation ? undefined : 'region',
```

Comment:
if we say that we are not setting the attribute otherwise, shouldn't we destructure these 3 attributes? Because how we are doing it now we are adding either the value or `undefined`, any of these options meaning that we are setting a value.

Reply:
also you can probabbly add the condition only once since it's the same one. something like
```js
...(props.navigation && { role: 'region', ... })
```